data/checks.yml

```diff
@@ -25,3 +25,10 @@ list:
   fail_title: Could not establish connection
   body: |
     We couldn't successfully connect to this mailserver to scan it. This could be an error on our side, too. If you're having trouble getting the scanner to work, shoot us an email at <a href="mailto:starttls-policy@eff.org">starttls-policy@eff.org</a>.
+- key: mtasts
+  title: Advertises MTA-STS policy
+  fail_title: Does not have MTA-STS policy up
+  body: |
+    <a href="/faq#mtasts">MTA-STS</a> is a new standard for protecting mailservers' TLS information from tampering (like STARTTLS downgrades) once a secure connection has been established in the past. If your server supports MTA-STS, other servers can remember and save your TLS information in the future and identify on-path attacks.
```

Comment:
I think "after a secure connection has been established for the first time" or "after the first time a secure connection is established".

Reply:
Also, it seems like maybe this is technically untrue because an active attacker could downgrade *the MTA-STS announcement itself* without downgrading individual STARTTLS banners, right? (Since MTA-STS policies are transmitted out-of-band with respect to individual underlying TLS sessions, unlike HSTS policies.)